zap/src/test/java/org/zaproxy/zap/extension/anticsrf/ExtensionAntiCSRFUnitTest.java

```diff
@@ -178,6 +178,30 @@ void shouldGetTokensFromInputFieldsIfIdIsAKnownToken() {
             assertAntiCsrfToken(tokens.get(2), message, KNOWN_TOKEN_3, "value3", 1);
         }
 
+        @Test
+        void shouldGetTokensFromInputFieldsIfKnownTokenWithPartialMatch() {
```

Comment:
This is already covered by `shouldGetTokensFromInputFieldsIfIdAndNameIncludesAKnownTokenAndPartialMatching*`. Though still worth adding one for `isAntiCsrfToken(String)` specifically.

Reply:
On it, will drop the println too :grinning: